client/my-sites/marketing/connections/services/index.js

```diff
@@ -1,12 +1,12 @@
-export { default as instagram } from './instagram';
+export { default as instagram_basic_display } from './instagram';
```

Comment:
should we change this to be kebab case?

Reply:
There's a certain amount of code that relies on these being snake case. We could fix that too, but that's maybe something for a larger refactor.